javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -1,9 +1,11 @@
 /* exported addSuffixToAll */
 function addSuffixToAll(words, suffix) {
+  var results = [];
   for (var i = 0; i < words.length; i++) {
     var temp = words[i];
     temp = temp + suffix;
     words[i] = temp;
```

Comment:
Almost! `words[i] = temp` is still changing the original, that line should be removed.